r_code/.gitignore

```diff
@@ -1 +1,2 @@
 *_Scratchwork.R
+gfoo_estimator.R
```

Comment:
Is this a temporary file?

Reply:
Yes, it was gitignored in my main branch but not on the Placebo matching branch, so I just added that line.